src/components/lab/canvas/IceCover.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Line } from 'react-konva';
+import {
+  FULL_MOUNTAIN,
+  ICE_COVER_FILL,
+  ICE_COVER_LINES_TENSION,
+} from '../../../config/constants';
+import {
+  generateFullMountainPoints,
+  generateHalfMountainPoints,
+} from '../../../utils/canvas';
+
+const IceCover = ({
+  mountainType,
+  iceCover,
+  mountainWidth,
+  mountainHeight,
+  mountainPeakX,
+  mountainPeakY,
+}) => {
+  const iceCoverHeight = iceCover * mountainHeight;
+  const iceCoverWidth = iceCover * mountainWidth;
+
+  const iceCoverBeginsX =
+    mountainType === FULL_MOUNTAIN
+      ? mountainPeakX - iceCoverWidth / 2
+      : mountainPeakX - iceCoverWidth;
+  const iceCoverBeginsY = mountainPeakY + iceCoverHeight;
+
+  const iceCoverPoints =
+    mountainType === FULL_MOUNTAIN
+      ? generateFullMountainPoints(iceCoverWidth, iceCoverHeight)
+      : generateHalfMountainPoints(iceCoverWidth, iceCoverHeight);
```

Comment:
Do you think it would be possible to use `scale` instead?

Reply:
Yes looks like this works, i.e. we just use the same `<Line />` component that defines the mountain, and give it a `scale={{}}` prop. I'll reconfigure.